Logging.Tests/LoggingTest.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Linq;
+using log4net.Config;
+using Logging.Entities;
+using NUnit.Framework;
+
+namespace Logging.Tests
+{
+    public class LoggingTest
+    {
+        private const string Hostname = "mockedHostname";
+        private const string Ip = "1.2.3.4";
+        private const string HttpRequestMethod = "POST";
+        private const string Path = "1.2.3.4/testRoute";
+        private const string Original = "Nunit Framework";
+        private const long Bytes = 42;
+        private const int StatusCode = 200;
+        private const float ResponseTime = 1337F;
+        
+        [Test]
+        public void TestIncomingRequestLog()
+        {
+            const string expectedLog =
+                "{\"Http\":{\"Request\":{\"Method\":\"" + HttpRequestMethod + "\"},\"Response\":null}," +
+                "\"Url\":" +
+                    "{\"Path\":\"" + Path + "\"}," +
+                    "\"UserAgent\":" +
+                        "{\"Original\":\"" + Original + "\"}," +
+                    "\"Host\":" +
+                        "{\"Hostname\":\"" + Hostname + "\",\"Ip\":\"" + Ip + "\"}}";
+            var mockRequest = BuildIncomingRequestLog();
+            var appender = new log4net.Appender.MemoryAppender();
+            BasicConfigurator.Configure(appender);
+            LoggingUtility.LogIncomingRequest(mockRequest);
+            
+            var result = appender.GetEvents();
+            Assert.IsTrue(result.Any());
+            Assert.IsTrue(result[0].MessageObject.Equals(expectedLog));
+        }
+        
+        [Test]
+        public void TestCompletedRequestLog()
+        {
+            var expectedLog =
```

Comment:
Should logs follow are guidelines?
https://docs.mia-platform.eu/development_suite/monitoring-dashboard/dev_ops_guide/log/

Fieldnames looks different from what specified in that guidelines

Reply:
I followed this example to compose log fields (Mandatory Logs): https://github.com/mia-platform/lc39/blob/master/lib/custom-logger.js#L43

Looking at the guidelines, fields have to be in lowercase, and the mandatory field "request_id" has to be added.